### ISSUE TITLE
Dynamic font size

### ISSUE THREAD

**Issue by ManuelWaldmann** [jean-emmanuel/open-stage-control#470]
Hello,

I created a surface with buttons, fader, etc, that adapts to the current resolution.
That works very well. Unfortunately, the font size always remains constant.
Especially if you switch from 1920x1080 to 800x600, that doesn't fit.
Is there a way to make font sizes dynamic?
In case, if the size of a button (fader,...) shrinks by half, the label also shrinks by half or quarter or other.
I tried a bit but nothing worked.

Thank you very much
Manu

**Comment by jean-emmanuel** [comment]
You just need to add some css to the root widget, for example this reduces the font size if the window/screen is smaller than 800px.

```css
@media screen and (max-width:800px) {
  :host {
    font-size: 80%;
  }
}
```

**Comment by ManuelWaldmann** [comment]
Thanks!

It is also possible if I only make the window smaller?
The buttons change the size. The text remains the same.
Also, the tab buttons retain their size.
The wide changes but the height always remains the same.
It is possible that the height changes with?

Best 
Manu

**Comment by jean-emmanuel** [comment]
Sorry but I don't understand your question, can you rephrase it ?

**Comment by ManuelWaldmann** [comment]
Here are pictures. So it's easier.
I hope you understand what I mean.

![big](https://user-images.githubusercontent.com/31275613/53006243-2169be80-3435-11e9-80bb-0e5473388d76.PNG)
original size. 
All is perfect.

![small](https://user-images.githubusercontent.com/31275613/53006978-a4d7df80-3436-11e9-9676-b44f2d6a729e.PNG)
quarter size
Everything is downsizing. Only what is marked is not reduced

**Comment by jean-emmanuel** [comment]
```css
@media screen and (max-width:800px) {
  :host {
    font-size: 80%;
    --nav-height:30rem;
  }
}
```

**Comment by ManuelWaldmann** [comment]
Great!
Thank you very much.
Now its all perfect.

You're the best!!

Manu

**Comment by jean-emmanuel** [comment]
Glad I could help, if you put your template online post a link and I'll add it to the examples.
